packages/test/snapshots/src/replayMultipleFiles.ts

```diff
@@ -103,8 +103,13 @@ export async function processOneNode(args: IWorkerArgs) {
     replayArgs.inDirName = args.folder;
     // The output snapshots to compare against are under "currentSnapshots" sub-directory.
     replayArgs.outDirName = `${args.folder}/${currentSnapshots}`;
-    replayArgs.snapFreq = args.snapFreq;
-
+    if (args.mode === Mode.NewSnapshots) {
+        // when generating new snapshots, match those from
+        // the original file based on summarize ops
+        replayArgs.testSummaries = true;
```

Comment:
What does this do? I am curious for my own understanding of the tests and replay tool.